### ISSUE TITLE
Welcome to Our Project! Share Your Ideas, Report Bugs, and Contribute 🚀

### ISSUE THREAD

**Issue by may-tas** [may-tas/TextEditingApp#1]
Hi everyone! 👋

Welcome to our project repository! We’re thrilled to have you here and would love to hear your thoughts, ideas, and feedback. This issue is a space to:

1. **Share Your Ideas** : Have a feature in mind? Let us know what you’d like to see in this project.
2. **Report Bugs** : Encountered an issue while using the project? Open a new issue with details so we can resolve it.
3. **Contribute** : Found an area to improve? Feel free to fork the repo, make changes, and submit a pull request (PR).

## How You Can Help:
1. Check the **README.md** for project details and guidelines.
2. Open a new issue to share ideas or report bugs.
3. Follow our **CONTRIBUTING.md** guidelines to submit a PR.

Your contributions make this project better! Let’s collaborate and build something amazing together. 💡✨
Thank you for being part of this journey! ❤️

**Comment by may-tas** [comment]
Hy @MannemSumanaSri This thread is for discussing and reporting any bugs or feature-requests. So I can't assign it to you.
Don't worry I will be opening some issues soon, you can comment on them and it will be assigned to you.

**Comment by may-tas** [comment]
Guys we are using Cubit as state management tool.

**Comment by may-tas** [comment]
@debasmitaas Yes you can open an issue. It could be a bug report or a feature-request.
I will also be adding some more issues soon.

**Comment by may-tas** [comment]
https://gssoc-tracker.netlify.app/projects-leaderboard

**Comment by Gupta-02** [comment]
Assigm me this
